en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -1456,6 +1456,9 @@ Follow the instructions below to move all the existing API Manager configuration
             ADD BILLING_CYCLE VARCHAR(15) DEFAULT NULL 
             ADD PRICE_PER_REQUEST VARCHAR(15) DEFAULT NULL 
             ADD CURRENCY VARCHAR(15) DEFAULT NULL
+            ADD MAX_COMPLEXITY INT NOT NULL DEFAULT 0
```

Comment:
Can we make all the ADD statements to the same indentation level

Reply:
fixed in 852da41efa31fc1c76a0f2990a926af244f9a8db